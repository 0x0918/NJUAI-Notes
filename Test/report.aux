\relax 
\providecommand\hyper@newdestlabel[2]{}
\providecommand\HyperFirstAtBeginDocument{\AtBeginDocument}
\HyperFirstAtBeginDocument{\ifx\hyper@anchor\@undefined
\global\let\oldcontentsline\contentsline
\gdef\contentsline#1#2#3#4{\oldcontentsline{#1}{#2}{#3}}
\global\let\oldnewlabel\newlabel
\gdef\newlabel#1#2{\newlabelxx{#1}#2}
\gdef\newlabelxx#1#2#3#4#5#6{\oldnewlabel{#1}{{#2}{#3}}}
\AtEndDocument{\ifx\hyper@anchor\@undefined
\let\contentsline\oldcontentsline
\let\newlabel\oldnewlabel
\fi}
\fi}
\global\let\hyper@last\relax 
\gdef\HyperFirstAtBeginDocument#1{#1}
\providecommand*\HyPL@Entry[1]{}
\HyPL@Entry{0<</S/D>>}
\HyPL@Entry{1<</S/D>>}
\@writefile{toc}{\contentsline {section}{\numberline {一、}全息投影简介}{2}{section.1}\protected@file@percent }
\@writefile{toc}{\contentsline {section}{\numberline {二、}商用的伪全息投影技术}{2}{section.2}\protected@file@percent }
\@writefile{toc}{\contentsline {subsection}{\numberline {1.}佩珀尔幻象}{3}{subsection.2.1}\protected@file@percent }
\@writefile{toc}{\contentsline {subsection}{\numberline {2.}旋转LED显示技术}{3}{subsection.2.2}\protected@file@percent }
\@writefile{toc}{\contentsline {subsection}{\numberline {3.}3D渲染}{3}{subsection.2.3}\protected@file@percent }
\@writefile{toc}{\contentsline {section}{\numberline {三、}全息投影技术的前沿发展}{4}{section.3}\protected@file@percent }
\bibcite{ref1}{1}
\@writefile{toc}{\contentsline {section}{\numberline {四、}总结}{5}{section.4}\protected@file@percent }
