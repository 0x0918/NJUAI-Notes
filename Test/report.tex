\documentclass[UTF8]{ctexart}
% 插入图片所使用的宏包
\usepackage{graphicx}
% 固定图片
\usepackage{float}
% 超链接和目录颜色
\usepackage[colorlinks, urlcolor=blue, linkcolor=black]{hyperref}
% 用于生成随机中文
\usepackage{zhlipsum}
% 设置section左对齐，默认是居中
% \CTEXsetup[format={\Large\bfseries}]{section}
\ctexset{ section = { format=\Large\bfseries } }
% 设置章节编号
\renewcommand\thesection{\chinese{section}、}
\renewcommand\thesubsection{\arabic{subsection}.}
\renewcommand\thesubsubsection{\alph{subsubsection})}
% 设置章节编号与章节标题的距离
\makeatletter
\renewcommand\@seccntformat[1]{%
    {\csname the#1\endcsname}\hspace{0.1em}
}
\makeatother

% 在这里设置你的标题和姓名
\title{全息投影的前世今生}
\author{赵思衡}

\begin{document}

% 标题页
\begin{titlepage}
    \newcommand{\HRule}{\rule{\linewidth}{0.5mm}}
    \center
    \quad \\[1.5cm]
    \textbf{\Huge 成像世界的奇遇结课论文 } \\[3cm]
    \makeatletter
    \HRule \\[0.8cm]
    { \huge \bfseries \@title} \\[0.4cm]
    \HRule \\[2cm]
    \begin{tabular}{rl}
        \textbf{\Large 院系:} & \textbf{\Large 人工智能学院}               \\[0.5cm]
        \textbf{\Large 姓名:} & \textbf{\Large \@author}                   \\[0.5cm]
        % 设定你的学号, 班级和邮箱
        \textbf{\Large 学号:} & \textbf{\Large 201300024}                  \\[0.5cm]
        \textbf{\Large 班级:} & \textbf{\Large 人工智能 20 级 2 班}        \\[0.5cm]
        \textbf{\Large 邮箱:} & \textbf{\Large 201300024@smail.nju.edu.cn} \\[0.5cm]
        % 默认为今天, 如果要修改为你需要的时间, 请将 \today 替换为你需要的时间
        \textbf{\Large 时间:} & \textbf{\Large \today}                     \\[2cm]
    \end{tabular}
    \vfill
\end{titlepage}

% 生成目录
\newpage
\tableofcontents
\newpage

% 加大段落间距
\setlength{\parskip}{1em}

% 主体内容
\section{全息投影简介}

目前大众所认知的全息投影有真全息投影和伪全息投影之分。

真全息投影即全息术（英语：Holography），又称全息投影、全息3D，是一种记录被摄物体反射（或透射）光波中全部信息（振幅、相位）的照相技术，而物体反射或者透射的光线可以通过记录胶片完全重建，仿佛物体就在那里一样。通过不同的方位和角度观察照片，可以看到被拍摄的物体的不同的角度，因此记录得到的像可以使人产生立体视觉。

其中“Holos”是希腊语“全部”的意思。Holography/Hologram（全息术/全息图）是匈牙利裔英国物理学家Dennis Gabor在1948年发明的，这篇开山之作发表在Nature上。

这个是真全息投影的准确定义，总结起来其实就是两点：

% 列表语法
\begin{enumerate}
    \item 一是需要裸眼，无介质，影像在空气中立体呈现；
    \item 二是可以从360度去观看立体影像的不同角度。
\end{enumerate}



\section{商用的伪全息投影技术}

既然已经知道了真全息投影的标准，那么我们就可以很容易的辨别什么是伪全息投影，什么是真全息投影了。目前谈到全息投影，我们想到最多的商用场景就是虚拟偶像演唱会，比如初音未來、洛天依。

但是很遗憾的是，它们都属于伪全息投影，并不是真正的全息投影。虽然虚拟偶像演唱会给我们呈现出了栩栩如生的立体影像，但是其必须在固定的舞台上，且要在黑暗当中才能实现，而且观众必须要从特定的角度进行观看。

以下是目前商用较多的几种伪全息投影：

\subsection{佩珀尔幻象}

虚拟偶像演唱会是怎么实现的呢？它实际上属于一种光学错觉技术，我们称之为佩珀尔幻象，在魔术表演中经常会用到。它的原理并不复杂，是利用一张半透半反的膜，也就是所谓的透明全息膜，作为介质，使得物体在膜中成了个虚像，因为是半透的，所以你可以看到膜后的景物，视觉上给人一种立体的错觉，再加上CG技术以及高亮度的灯光，这种立体影像就会给观众一种惟妙惟肖的真实感觉。

目前商用领域所谓的全息投影大多都是利用的佩珀尔幻象的原理，并不是真正的全息投影。真正的全息投影技术其实还有很多技术门槛需要被攻破，目前只能停留在实验室阶段。

全息投影技术的实现本质上和现在的电影放映技术是一样的，都是对“光”的控制。先采集所需要的内容信息，再复原这些信息，只不过电影放映技术是采集并复原平面信息，而全息投影是要采集并复原立体信息。电影放映技术是用幕布作为介质来承载内容，而目前全息投影的立体信息还没有一个成本低且稳定的介质，可以承载这些立体信息。

\subsection{旋转LED显示技术}

旋转LED显示技术看着像裹了一圈LED的风扇，转起来后会有3D视觉效果。这种技术利用了视觉暂留原理，通过LED的高速旋转来实现平面成像，但由于LED灯条在旋转时并非密不透风，观察者依然可以看到灯条后的物体，从而让观察者感觉画面悬浮在空中，实现类似3D的效果。有人叫他“3D全息风扇屏”，但这个也和“全息”没有任何关系。

\subsection{3D渲染}

3D渲染即增强现实技术（Augmented Reality，简称AR），也有对应VR虚拟实境一词的翻译称为实拟虚境或扩张现实，是指透过摄影机影像的位置及角度精算并加上图像分析技术，让屏幕上的虚拟世界能够与现实世界场景进行结合与交互的技术。3D渲染是后期人为加的3D效果，是一种视频处理手段，现实中看不到。通过后期特效与拍摄画面的合成达到效果。现场观众可以通过大屏幕观看。优点是对场地没有要求，和舞台上其他人互动性强，视角灵活，可搭配其他特效(比如变换场景之类的)。缺点自然就是无法直接看到，只能通过屏幕。


\section{全息投影技术的前沿发展}

目前全球已知的全息投影技术有三种，分别是360度全息显示屏技术、空气投影技术、激光束投射技术。其中360度全息显示屏技术最容易理解，它是将图像投射镜子上，再让镜子进行高速的旋转，从而产生3D的立体影像；空气投影技术则是利用水蒸气，将影像投射在水蒸气上，由于分子之间的震动不均衡，所以可以形成立体图像；激光束投射技术是最为复杂的，它是利用氮气和氧气在空气中散开时，混合成的气体变成灼热的浆状物质，并在空气中投射出3D影像，但这种技术显示的时间很短暂。

而现在，杨百翰大学（Brigham Young University）的研究者们，则通过激光束捕捉物理粒子，创造出了真正的能够漂浮在空气中的，动态的立体图像：想象在一个充满灰尘的房间中，用强光一照，你就能看到飞舞的灰尘反射光线，在空气中形成许多小亮点。利用激光来照射实体粒子并使其向四处反光也同理。而现在，如果我们能控制这个粒子的轨迹，并且让这个粒子在这个轨迹上进行极快的周期性运动（scan），那么此时粒子反光的轨迹就会形成一个立体图像。

这就是OTD（Optical Trap Display），\textbf{光学捕捉显示}技术。也是BYU电机工程学教授Dan Smalley和他的团队在这个项目里所使用的核心技术。

他们先使用激光束捕捉空气中的微小物理粒子，然后快速移动。当粒子被拖曳穿过空间时，可见光会通过激光束将其照亮，形成一条运动路径。而如果这时以高于眼睛的闪烁速率每秒重新绘制10次以上，就能通过视觉的持久性来形成图像。并且，在粒子的移速足够快时，其位置和颜色都可以被改变，从而形成颜色各异的动态体积图像（volume image）。

除此之外，研究团队还利用透视投影（Perspective Projection）技术，随着观看者视角的移动来修改图像的边缘及视差，在背景生成模拟的非体积图像点，以增加对图像体积或深度的感知。这时，你就完全可以环绕图像一圈，360度无死角地去观察它了。这些自由浮动的全息图像，本身是在固定体积大小的空间中，由激光束捕捉粒子构建的，所以只能生成微小的3D全息图。团队里的Wesley Rogers表示，如果要构建一座真实大小的山峰模型，那也必须有一个体积相同甚至更大的空间，来捕捉这整个空间中的粒子。这显然是不现实的，于是他们使用了一些视觉技巧，如运动视差（Motion Parallax）技术，来使场景中移动的图像在显示时，看起来比实际要大得多。而对比大多数还是要求观众盯着屏幕展现效果的3D投影，这项技术所展现的物理的，而非幻象的投影，真正做到了幻觉与真人的互动。


\section{总结}

综上所述，目前真正的全息投影大多只能限制于实验室的环境下，无法做到大规模的商用。而商用的所谓全息投影其实都不是真正的全息投影，它们无法脱离成像的介质，只能依附于某种成像镜子或膜来进行投影。而我们离真正的全息投影技术还有很远的路要走。

\begin{thebibliography}{99}  
\bibitem{ref1}Rogers W, Smalley D. Simulating virtual images in optical trap displays[J]. Scientific reports, 2021, 11(1): 1-6.
\end{thebibliography}


\end{document}