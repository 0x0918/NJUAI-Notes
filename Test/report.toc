\contentsline {section}{\numberline {一、}全息投影简介}{2}{section.1}%
\contentsline {section}{\numberline {二、}商用的伪全息投影技术}{2}{section.2}%
\contentsline {subsection}{\numberline {1.}佩珀尔幻象}{3}{subsection.2.1}%
\contentsline {subsection}{\numberline {2.}旋转LED显示技术}{3}{subsection.2.2}%
\contentsline {subsection}{\numberline {3.}3D渲染}{3}{subsection.2.3}%
\contentsline {section}{\numberline {三、}全息投影技术的前沿发展}{4}{section.3}%
\contentsline {section}{\numberline {四、}总结}{5}{section.4}%
