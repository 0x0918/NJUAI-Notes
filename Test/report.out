\BOOKMARK [1][-]{section.1}{\376\377\121\150\140\157\142\225\137\161\173\200\116\313}{}% 1
\BOOKMARK [1][-]{section.2}{\376\377\125\106\165\050\166\204\117\052\121\150\140\157\142\225\137\161\142\200\147\057}{}% 2
\BOOKMARK [2][-]{subsection.2.1}{\376\377\117\151\163\300\134\024\136\173\214\141}{section.2}% 3
\BOOKMARK [2][-]{subsection.2.2}{\376\377\145\313\217\154\000L\000E\000D\146\076\171\072\142\200\147\057}{section.2}% 4
\BOOKMARK [2][-]{subsection.2.3}{\376\377\0003\000D\156\062\147\323}{section.2}% 5
\BOOKMARK [1][-]{section.3}{\376\377\121\150\140\157\142\225\137\161\142\200\147\057\166\204\122\115\154\277\123\321\134\125}{}% 6
\BOOKMARK [1][-]{section.4}{\376\377\140\073\176\323}{}% 7
